modules/nw-ingress-creating-a-route-via-an-ingress.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * ingress/routes.adoc
+
+[id="nw-ingress-creating-a-route-via-an-ingress_{context}"]
+= Creating a Route via an Ingress
+
+Some ecosystem components have an integration with Ingress resources but not with
+Route resources. To cover this case, {product-title} automatically creates
+managed Route objects when an Ingress object is created. These Route objects are deleted
+when the corresponding Ingress objects are deleted.
+
+.Procedure
+
+Define an Ingress object:
++
+.YAML Definition of an Ingress
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: frontend
+  annotations:
+    route.openshift.io/termination: "reencrypt" <1>
+spec:
+  rules:
+  - host: www.example.com
+    http:
+      paths:
+      - backend:
+          serviceName: frontend
+          servicePort: 443
+  tls:
+  - hosts:
+    - www.example.com
+    secretName: example-com-tls-certificate
+----
++
+
+<1> The `route.openshift.io/termination` annotation can be used to configure the `spec.tls.termination` field of the `Route`
+as `Ingress` has no field for this. The accepted values are `edge`, `passthrough` and `reencrypt`. All
+other values are silently ignored. When unset, `edge` is used.
+
+Apply it to your cluster:
++
+----
```

Comment:
Add [source,terminal] before the code block (this is a newer style change throughout the docs).

e.g.

. Apply the YAML file to your cluster:
+
[source,terminal]
...
```suggestion
----
```